src/main/java/io/cdap/plugin/dlp/configs/DlpFieldTransformationConfig.java

```diff
@@ -177,6 +178,12 @@ public ErrorConfig getErrorConfig(String transformPropertyId) {
     }
     return names;
   }
+
+  public HashSet getRequiredFields() {
```

Comment:
Use Set and always have the parameter type

Reply:
done